tests/remote.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from "node:test";
+
+import cw, { morgen } from "..";
+const { log, fetchMorgen } = morgen.util;
+const { luxon } = morgen.deps;
+
+// NOTE: These tests require setting MORGEN_API_KEY and the Workflow should be
+// configured to have a calendar account (via the UI because it's the only way)
```

Comment:
Couldn't we update the workflow configs via the `https://api.morgen.so/workflows/update` endpoint? I think it would be nice if this test runs without requiring any manual interaction, also deleting the workflow as part of the test clean up at the end. But maybe this can be done as part of a later iterartion.

Reply:
Yep, totally agree it should be fully automated.

Yep, I think we should have a dedicated E2E Morgen user and put the API key for that somewhere (GSM maybe?) when we do want to make this production ready.

I think for now it's fine that it's relatively manual and least it throws errors when something isn't configured properly.